### PR TITLE
The TestCase for ShenyuPluginLoader.

### LINKED ISSUE #2078: [Volunteer] Add unit test for ShenyuPluginLoader
### Volunteer help

Hi community

Volunteer help to add unit test for `org.apache.shenyu.web.loader.ShenyuPluginLoader`

### PR DESCRIPTION
**Pull request by tonytanCoder**
// The TestCase for ShenyuPluginLoader.. Fixes #2078

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by yu199195** [comment]
pls check ci。

File: shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginLoader.java [hunk 1]
```diff
@@ -30,11 +30,7 @@
 import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Enumeration;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
```
Comment (tydhot) [inline shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginLoader.java:33]: plz don't use * in imports

**Comment by yu199195** [comment]
@tonytanCoder  can you check ci？it not pass~

**Comment by midnight2104** [comment]
hi @tonytanCoder please check ci.

**Comment by songyuequan** [comment]
hi @tonytanCoder please check ci.

**Review by songyuequan — CHANGES_REQUESTED** [review]
Hi~, please check ci .

**Comment by JooKS-me** [comment]
Done in #2623 .

**Pull request closed without merging**